boards/posix/native_posix/doc/board.rst

```diff
@@ -362,11 +369,51 @@ The following peripherals are currently provided with this board:
   Normally the Zephyr application and HW models run in very little time
   on the host CPU, so this is a good enough approach.
 
-.. _NATIVE_POSIX_SLOWDOWN_TO_REAL_TIME:
-   ../../../../reference/kconfig/CONFIG_NATIVE_POSIX_SLOWDOWN_TO_REAL_TIME.html
-
 **Interrupt controller**:
   A simple yet generic interrupt controller is provided. It can nest interrupts
   and provides interrupt priorities. Interrupts can be individually masked or
   unmasked. SW interrupts are also supported.
 
+Shell support
+*************
+
+When the :ref:`Shell` subsystem is compiled with your application, the native
+standard input (`stdin`) will be redirected to the shell.
+You may use the shell interactively through the console,
+by piping another process output to it, or by feeding it a file.
+
+When using it interactively you may want to select the option
+:option:`CONFIG_NATIVE_POSIX_SLOWDOWN_TO_REAL_TIME`.
+
+When feeding `stdin` from a pipe or file, the console driver will ensure
+reproducibility between runs of the process:
+
+  The execution of the process will be stalled while waiting for new `stdin`
+  data to be ready.
+
+  Commands will be feed to the shell as fast as the shell can process them.
+  To allow controlling the flow of commands to the shell, a special command has
+  been added ``!wait <ms>``.
```

Comment:
Do you need to update the Shell documentation (in doc/subsystems/shell.rst ) to include a description for this command in the http://docs.zephyrproject.org/subsystems/shell.html#using-shell-commands section?  Perhaps this ``!wait`` is not actually a Shell command (since you mention it's handled by the console driver).

Reply:
As you mention, it is not really a shell command. I did not know how to call it other than "special command", as it really only affects the driver itself, and won't be seen by the shell.
So I think it may be a bit confusing to add it to the shell subsystem documentation.

Reply:
As you've clarified, I agree it shouldn't be documented as a shell command.  Maybe call it a "driver directive" or is that too odd?

Reply:
"Driver directive" is probably better. I will change it that way.
(But if later some better name pops to your mind just tell and I will update it)

Reply:
I rephrased a bit this paragraph. Please tell if it is clearer now, or if some changes would make it better. 
Like always, thanks you very much.